### PR TITLE
Support arbitrary non-transformed Github URL pages in sitemap

### PR DESCRIPTION
**Pull request by varunarora**

### CODE REVIEW

File: portal/deploy/documentation.py [hunk 1]
```diff
@@ -45,3 +49,16 @@ def transform(source_dir, version, output_dir, specified_source=None):
             convertor(extracted_source_dir, version, settings.EXTERNAL_TEMPLATE_DIR)
         else:
             return
+
+
+def fetch_and_transform(source_url, version):
+    response = requests.get(source_url)
+    source_markdown_file = '/tmp' + urlparse(source_url).path
```
Comment (jetfuel) [inline portal/deploy/documentation.py]: or you can use tempfile.gettempdir()

Comment (varunarora) [reply portal/deploy/documentation.py]: Oh great idea! However, I need to strip is out in the next step, but I don't know how I will be able to do that if I don't know the tmp path.

Comment (cs2be) [inline portal/deploy/documentation.py]: shoudl use tempfile (import tempfile)

File: portal/deploy/documentation.py [hunk 2]
```diff
@@ -45,3 +49,16 @@ def transform(source_dir, version, output_dir, specified_source=None):
...
+
+    if not os.path.exists(os.path.dirname(source_markdown_file)):
+        os.makedirs(os.path.dirname(source_markdown_file))
+
+    with open(source_markdown_file, 'wb') as f:
+        f.write(response.content)
+
+    strip.markdown_file(source_markdown_file, version)
```
Comment (cs2be) [inline portal/deploy/documentation.py]: Does images work?

File: portal/deploy/strip.py [hunk 3]
```diff
@@ -162,3 +164,24 @@ def models(original_documentation_dir, version, destination_documentation_dir):
 
             elif 'images' in subpath:
                 copyfile(os.path.join(subdir, file), new_path)
+
+
+def markdown_file(source_markdown_file, version):
```
Comment (cs2be) [inline portal/deploy/strip.py]: Should make this a generic method.  If its only for Mobile, should rename method to reflect that

Comment (cs2be) [reply portal/deploy/strip.py]: Also, when does the content refresh, when we restart server?

**Comment by cs2be** [comment]
Otherwise LGTM

File: portal/deploy/documentation.py (hunk 2 above)
Comment (varunarora) [reply portal/deploy/documentation.py]: Not yet :(

File: portal/deploy/strip.py (hunk 3 above)
Comment (varunarora) [reply portal/deploy/strip.py]: The content doesn't refresh unless the old content is thrown. So, good question. I wonder how to handle this.

**Comment by jetfuel** [comment]
LGTM

**Pull request merged**